src/scripts/rx-data-table.src.js

```diff
@@ -295,8 +295,9 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
                         return editable.options;
                     }
                 }
-
-                return [];
+                else {
```

Comment:
This else is formatted inconsistently.  It also appears to create a path where nothing would get returned.  Not sure if that is what your going for.